source/documentation/other-topics/access-cross-aws-resources.html.md.erb

```diff
@@ -0,0 +1,259 @@
+      role       = aws_iam_role.<namespace>-<role-name>.name
+      policy_arn = aws_iam_policy.<namespace>-<policy-name>.arn
+    }
+    ```
+    </p>
+    </details>
+
+    <details><summary> Click to see an example code block</summary>
+    <p>
+
+
+    ```
+    data "aws_iam_policy_document" "my-namespace-kiam-trust-chain" {
+    # KIAM trust chain to allow pods to assume roles defined below
+    statement {
+      principals {
+        type        = "AWS"
+        identifiers = [data.aws_iam_role.nodes.arn]
+      }
+      actions = ["sts:AssumeRole"]
+    }
+    }
+
+    variable "my-namespace-tags" {
+    type = map(string)
+    default = {
+      business-unit          = "Cloud Platform"
+      application            = "My Application"
+      is-production          = "false"
+      environment-name       = "Development"
+      owner                  = "cloud-platform"
+      infrastructure-support = "platforms@digital.justice.gov.uk"
+    }
+    }
+
+    resource "aws_iam_role" "my-namespace-app-iam-role" {
+    name               = "my-namespace-app-iam-role"
+    description        = "IAM role for My app to access s3 bucket - AAA in 11111 AWS account"
+    tags               = var.my-namespace-tags
+    assume_role_policy = data.aws_iam_policy_document.my-namespace-kiam-trust-chain.json
+    }
+
+    resource "kubernetes_secret" "iam-role-creds-my-app" {
+    metadata {
+      name      = "iam-role-creds-my-app"
+      namespace = "my-namespace"
+    }
+
+    data = {
+      arn       = aws_iam_role.my-namespace-app-iam-role.arn
+      name      = aws_iam_role.my-namespace-app-iam-role.name
+      unique_id = aws_iam_role.my-namespace-app-iam-role.unique_id
+    }
+    }
+
+    data "aws_iam_policy_document" "my-namespace-app-iam-role" {
+
+    # allow pods to assume this role
+    statement {
+      actions   = ["sts:AssumeRole"]
+      resources = [aws_iam_role.my-namespace-app-iam-role.arn]
+    }
+
+    # Provide list of permissions and target AWS account resources to allow access from
+    statement {
+      actions = [
+        "s3.PutObject",
+      ]
+      resources = [
+        "arn:aws:s3:::AAAAAAAA/AA-bucket/folder/*",
+      ]
+    }
+    }
+
+    resource "aws_iam_policy" "my-namespace-app-iam-policy" {
+    name   = "my-namespace-app-iam-policy"
+    policy = data.aws_iam_policy_document.my-namespace-app-iam-policy.json
+    }
+
+    resource "aws_iam_role_policy_attachment" "my-namespace-app-iam-policy" {
+    role       = aws_iam_role.my-namespace-app-iam-role.name
+    policy_arn = aws_iam_policy.my-namespace-app-iam-policy.arn
+    }
+    ```
+    </p>
+    </details>
+
+
+2. **Annotate the pod with the IAM role**
+
+    Once the PR in **step 1** is merged and applied, you will get the role ARN, name and id as a kubernetes secret in your namespace.
+
+    Decode the role ARN and annotate the pod to indicate which role which will need access to resource in different AWS account like below:
+
+    ```
```

Comment:
Add something here to tell the reader that this snippet needs to go into their deployment.yaml file, and a bit more context around the two lines so that they know **where** in that file to put this code.